tests/expected.yaml

```
@@ -152,29 +154,29 @@ spec:
           image: "camptocamp/sub@sha256:1234"
           imagePullPolicy: IfNotPresent
           env:
-              - name: "comfigmap"
-                valueFrom:
-                  secretKeyRef:
-                    name: "s"
-                    key: "data"
-              - name: "configmap"
-                valueFrom:
-                  configMapKeyRef:
-                    name: "cm"
-                    key: "data"
-              - name: "configmapoverride"
-                valueFrom:
-                  configMapKeyRef:
-                    name: "overitten"
-                    key: "data"
-              - name: "direct"
-                value: "test"
-              - name: "directvalue"
-                value: "test"
-              - name: "zz_first"
-                value: "zz"
-              - name: "aa_last"
-                value: "aa$(zz_first)aa"
+            - name: "comfigmap"
```

```suggestion
            - name: "configmap"
```

But this will result in duplicated env key configmap.
And it seems wreid having env vars name with lower case.

---

Effectively, thanks, it should be secret :-)